src/EventNames.ts

```diff
@@ -0,0 +1,7 @@
+/**
+* Eventname class for use instead of hardcoded strings
+*/
+export default class EventNames {

```

Comment:
I dislike `default` in exports. It makes it more difficult to extend the file later on to export multiple things.